templates/wp-config.mustache

```diff
@@ -75,8 +75,9 @@ define( 'WPLANG', '{{locale}}' );
 /* That's all, stop editing! Happy publishing. */
 
 /** Absolute path to the WordPress directory. */
-if ( ! defined( 'ABSPATH' ) )
+if ( ! defined( 'ABSPATH' ) ){
 	define( 'ABSPATH', dirname( __FILE__ ) . '/' );
+}
 
 /** Sets up WordPress vars and included files. */
-require_once ABSPATH . 'wp-settings.php';
+require_once( ABSPATH . 'wp-settings.php' );
```

Comment:
Is that nonsense still a thing in Core? I thought everything ran through PHPCS... ?

Reply:
Yes, it still there :)
See here: https://github.com/WordPress/WordPress/blob/master/wp-config-sample.php#L90

Reply:
Maybe it needs a ticket to fix!

Reply:
Can you remove the parentheses again? I don't think I want WP-CLI to further encourage this and add to the confusion.